README.md

```diff
@@ -43,12 +45,42 @@ It deploys a hello-openshift application 3 times:
 
 
 
-## Bootstrapping
+## Setup instructions
+
+### Default single cluster setup with `values-simple.yaml`
+
+#### Configuring required secrets / parameters
+The secrets here secure Trustee and the peer-pod vms. Mostly they are for demonstration purposes. 
+This only has to be done once.
 
-#### Install of OCP cluster on azure.
+1. `cp ./values-secret.yaml.template ~/values-secret-coco-pattern.yaml`
+2. Examine the sample secrets e.g. `kbsRes1` and `passphrase` and set as appropriate or random secrets will be generated in vault
+3. If you have not previously, run the following:
```

Comment:
Maybe it would be simpler to have a single shell script? One that will skip things if they are already present?